ci-operator/step-registry/ipi/conf/azurestack/ipi-conf-azurestack-commands.sh

```diff
@@ -56,3 +56,17 @@ EOF
 
 echo "${AZURESTACK_ENDPOINT}" >> ${SHARED_DIR}/AZURESTACK_ENDPOINT
 echo "${SUFFIX_ENDPOINT}" >> ${SHARED_DIR}/SUFFIX_ENDPOINT
+APP_ID=$(jq -r .clientId "${SHARED_DIR}/osServicePrincipal.json")
+AAD_CLIENT_SECRET=$(jq -r .clientSecret ${SHARED_DIR}/osServicePrincipal.json)
+TENANT_ID=$(jq -r .tenantId "${SHARED_DIR}/osServicePrincipal.json")
+cat >> "${SHARED_DIR}/azurestack-login-script.sh" << EOF
+az cloud register \
+    -n PPE \
```

Comment:
nit: the name `PPE` refers to a specific environment. We could either make this a variable or something more generic.